CraftTweaker2-MC1120-Main/src/main/java/crafttweaker/mc1120/player/MCPlayer.java

```diff
@@ -39,6 +39,14 @@ public String getName() {
         return player.getName();
     }
     
+    @Override
+    public String getUUID() {
+        String uuid = EntityPlayer.getUUID(player.getGameProfile()).toString().toLowerCase();
+        if (uuid != null && !uuid.equals(""))
```

Comment:
uuid can never be `null` here because if `EntityPlayer.getUUID(player.getGameProfile())` were to return `null` you'd already get a NPE in line 44.

Maybe use something like

```java
UUID uuid = EntityPlayer.getUUID(player.getGameProfile());
if(uuid == null) {
    uuid = EntityPlayer.getOfflineUUID(player.getName());
}
if(uuid == null) {
    CrafttweakerAPI.logError("Could not get UUID for player " + getName());
    return "";
}

return uuid.toString().toLowerCase();
```

Also, since I don't know these vanilla methods that well, can `player.getGameProfile` return `null`, and if it can, will `getUUID(null)` throw an error?